fmt@11.0.2.bcr.1

jwnimmer-tri opened the pull request:
Add module dependencies on rules_cc and rules_license.
Use cc_library from rules_cc instead of native.
Add license annotations.
Test with Bazel 8.x and rolling (in addition to 7.x) and ubuntu2404.

bazel-io commented:
Hello @Vertexwahn, modules you maintain (fmt) have been updated in this PR. Please review the changes.

Vertexwahn commented on modules/fmt/11.0.2.bcr.1/presubmit.yml:
```diff
@@ -0,0 +1,26 @@
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
```
Please add `ubunut2404

jwnimmer-tri committed:
fmt@11.0.2.bcr.1

Add module dependencies on rules_cc and rules_license.
Use cc_library from rules_cc instead of native.
Add license annotations.
Test with Bazel 8.x and rolling (in addition to 7.x) and ubuntu2404.

bazel-io reviewed (approved):
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.
